plugins/services/src/js/service-configuration/PodContainerConfigSection.js

```diff
@@ -45,36 +65,42 @@ module.exports = ({containerConfig, appConfig}) => {
         <ConfigurationMapLabel>Container Image</ConfigurationMapLabel>
         <ConfigurationMapValueWithDefault
           value={findNestedPropertyInObject(appConfig, 'image.id')} />
+        {editLink}
```

Comment:
You are hitting the nail on the head here. The problem was I didn't have access to props in other components, so I opted for creating it outside, but this has since changed in the initial PR.